Can't start Slay the Spire

Slay the Spire refuses to start with the following error:

```
Exception in thread "LWJGL Application" java.lang.ExceptionInInitializerError
	at com.badlogic.gdx.backends.lwjgl.LwjglGraphics.setVSync(LwjglGraphics.java:558)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:124)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
	at org.lwjgl.opengl.LinuxDisplay.getAvailableDisplayModes(LinuxDisplay.java:954)
	at org.lwjgl.opengl.LinuxDisplay.init(LinuxDisplay.java:738)
	at org.lwjgl.opengl.Display.<clinit>(Display.java:138)
	... 2 more
AL lib: (EE) alc_cleanup: 1 device not closed
```
According to issues created for the Steam flatpak, the xrandr utility is necessary in order for the game to run.
https://github.com/flathub/com.valvesoftware.Steam/issues/619
https://github.com/flathub/com.valvesoftware.Steam/issues/121


I checked and I confirm that the xrandr utility is not in the image:

```
laurent ~ $ flatpak enter io.github.sharkwouter.Minigalaxy bash
bash-5.1$ find / -name xrandr
find: '/dev/vboxusb': Permission denied
find: '/proc/tty/driver': Permission denied
bash-5.1$ 
```

--- a/generated/libs.json
+++ b/generated/libs.json
@@ -184,6 +184,46 @@
                     }
                 }
             }
+        },
+        {
+            "name": "xrandr",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.1.tar.xz",
+                    "sha256": "7bc76daf9d72f8aff885efad04ce06b90488a1a169d118dea8a2b661832e8762"
+                }
+            ]
+        },
+        {
+            "name": "xrandr-32bit",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.1.tar.xz",
+                    "sha256": "7bc76daf9d72f8aff885efad04ce06b90488a1a169d118dea8a2b661832e8762"
+                }
+            ],
+            "only-arches": [
+                "x86_64"
+            ],
+            "build-options": {
+                "prepend-ld-library-path": "/app/lib32",
+                "prepend-pkg-config-path": "/app/lib32/pkgconfig",
+                "ldflags": "-L/app/lib32 -Wl,-z,relro,-z,now -Wl,--as-needed",
+                "ldflags-override": true,
+                "libdir": "/app/lib32",
+                "arch": {
+                    "x86_64": {
+                        "append-path": "/usr/lib/sdk/toolchain-i386/bin",
+                        "prepend-pkg-config-path": "/usr/lib/i386-linux-gnu/pkgconfig",
+                        "env": {
+                            "CC": "ccache i686-unknown-linux-gnu-gcc",
+                            "CXX": "ccache i686-unknown-linux-gnu-g++"
+                        }
+                    }
+                }
+            }
         }
     ]
 }

--- a/modules/libs.yml
+++ b/modules/libs.yml
@@ -34,3 +34,9 @@ modules:
           - type: archive
             url: https://libbsd.freedesktop.org/releases/libmd-1.0.4.tar.xz
             sha256: f51c921042e34beddeded4b75557656559cf5b1f2448033b4c1eec11c07e530f
+
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.1.tar.xz
+        sha256: 7bc76daf9d72f8aff885efad04ce06b90488a1a169d118dea8a2b661832e8762

